keyboards/v60_type_r/keymaps/default/keymaps/greenshadowmaker/keymap.c

```diff
@@ -0,0 +1,136 @@
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
+
+#define _QWERTY 0
+#define _LOWER 1
+#define _RAISE 2
+#define _ADJUST 16
```

Comment:
I see you fixed this stuff in your ergodash keymap, but looks like you missed this one ;)
Specifically, the layer numbers, trailing backslashes, and `break` after `return`.